src/rogue/interfaces/stream/TcpCore.cpp

```diff
@@ -131,10 +131,15 @@ ris::TcpCore::TcpCore (std::string addr, uint16_t port, bool server) {
 
 //! Destructor
 ris::TcpCore::~TcpCore() {
-  this->close();
+  this->stop();
 }
 
+// deprecated
 void ris::TcpCore::close() {
+   this->stop();
```

Comment:
Need to add a print statement about how  `close()` is deprecated and will be removed in the future and should use `stop()` instead